client.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Twitch Interactive, Inc.  All Rights Reserved.
```

Comment:
maybe rename the files to `client_hooks.go` / `client_hooks_test.go` for consistency with the server-side ones?